h/static/scripts/app-controller.js

```diff
@@ -46,37 +60,21 @@ module.exports = function AppController(
 
   // Reload the view when the user switches accounts
   $scope.$on(events.USER_CHANGED, function (event, data) {
+    $scope.auth = authStateFromUserID(data.userid);
+    $scope.accountDialog.visible = false;
+
     if (!data || !data.initialLoad) {
       $route.reload();
     }
   });
 
-  identity.watch({
-    onlogin: function (identity) {
-      // Hide the account dialog
-      $scope.accountDialog.visible = false;
-      // Update the current logged-in user information
-      var userid = auth.userid(identity);
-      var parsed = parseAccountID(userid);
-      angular.copy({
```

Comment:
Yes. `$scope.auth` is now purely data derived from `session.state.userid` which is passed down to `<signin-control>` and `<group-list>` for controlling which parts of those components are displayed. In `signin_control.js` and `group_list.js`, those are already (prior to this commit) using one-way (`<`) bindings to indicate that auth is only an input property which is not modified by those components.